test/common_tester.h

```diff
@@ -664,7 +765,24 @@ class JacobianTester : public ::testing::Test
 
     Adj = state.adj();
 
-    const Tangent tan = state.lift();
+    Tangent tan = state.lift();
+
+    Jr = tan.rjac();
+    Jl = tan.ljac();
+
+    EXPECT_EIGEN_NEAR(Jl, Adj*Jr);
+    EXPECT_EIGEN_NEAR(Adj, Jl*Jr.inverse());
+
+    // Jr(-tau) = Jl(tau)
+
+    EXPECT_EIGEN_NEAR(Jl, (-tan).rjac());
+
+    /////
```

Comment:
@joansola Duplicated test for the group element == Id to test the small angle(s) approximations.